From 9f5ff985d768693f5fedc7277c94b6c69eb010f4 Mon Sep 17 00:00:00 2001
From: Felix Dreissig <felix.dreissig@fau.de>
Date: Tue, 18 May 2021 17:57:56 +0000
Subject: [PATCH 24/30] Update mangled names

---
 .../Transforms/Utils/Enclavization/EnclaveModify.cpp   | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/llvm/lib/Transforms/Utils/Enclavization/EnclaveModify.cpp b/llvm/lib/Transforms/Utils/Enclavization/EnclaveModify.cpp
index ef2af0ac0a5a..34b68cd3aadf 100644
--- a/llvm/lib/Transforms/Utils/Enclavization/EnclaveModify.cpp
+++ b/llvm/lib/Transforms/Utils/Enclavization/EnclaveModify.cpp
@@ -101,7 +101,7 @@ static StructType *getBufreaderSgxfileType(Module *mod) {
 }
 
 
-#define MANGLED_SGXFS_OPENOPTIONS_NEW_NAME "_ZN8sgx_tstd5sgxfs11OpenOptions3new17hf4c8178c8abfed24E"
+#define MANGLED_SGXFS_OPENOPTIONS_NEW_NAME "_ZN8sgx_tstd5sgxfs11OpenOptions3new17hd1e80d08f14e9f75E"
 
 static Function *getSgxfsOpenoptionsNewDecl(Module *mod) {
   Function *func = mod->getFunction(MANGLED_SGXFS_OPENOPTIONS_NEW_NAME);
@@ -117,7 +117,7 @@ static Function *getSgxfsOpenoptionsNewDecl(Module *mod) {
 }
 
 
-#define MANGLED_SGXFS_OPENOPTIONS_READ_NAME "_ZN8sgx_tstd5sgxfs11OpenOptions4read17hb91f6c1788ad394aE"
+#define MANGLED_SGXFS_OPENOPTIONS_READ_NAME "_ZN8sgx_tstd5sgxfs11OpenOptions4read17h110ab21457ce6e1fE"
 
 static Function *getSgxfsOpenoptionsReadDecl(Module *mod) {
   Function *func = mod->getFunction(MANGLED_SGXFS_OPENOPTIONS_READ_NAME);
@@ -134,7 +134,7 @@ static Function *getSgxfsOpenoptionsReadDecl(Module *mod) {
 }
 
 
-#define MANGLED_SGXFS_OPENOPTIONS_WRITE_NAME "_ZN8sgx_tstd5sgxfs11OpenOptions5write17h5b10a1808d5f12e5E"
+#define MANGLED_SGXFS_OPENOPTIONS_WRITE_NAME "_ZN8sgx_tstd5sgxfs11OpenOptions5write17h22d925a63b24b91bE"
 
 static Function *getSgxfsOpenoptionsWriteDecl(Module *mod) {
   Function *func = mod->getFunction(MANGLED_SGXFS_OPENOPTIONS_WRITE_NAME);
@@ -151,7 +151,7 @@ static Function *getSgxfsOpenoptionsWriteDecl(Module *mod) {
 }
 
 
-#define MANGLED_SGXFS_OPENOPTIONS_APPEND_NAME "_ZN8sgx_tstd5sgxfs11OpenOptions6append17h14079cb0dd2caf72E"
+#define MANGLED_SGXFS_OPENOPTIONS_APPEND_NAME "_ZN8sgx_tstd5sgxfs11OpenOptions6append17hd2be1b734db32c6aE"
 
 static Function *getSgxfsOpenoptionsAppendDecl(Module *mod) {
   Function *func = mod->getFunction(MANGLED_SGXFS_OPENOPTIONS_APPEND_NAME);
@@ -252,7 +252,7 @@ static Function *getCadoteSgxfileReadtoendDecl(Module *mod, Type *usizeResultTyp
 }
 
 
-#define MANGLED_IO_WRITE_WRITEALL_NAME "_ZN8sgx_tstd2io5Write9write_all17h3e2682f553f4eb0fE"
+#define MANGLED_IO_WRITE_WRITEALL_NAME "_ZN8sgx_tstd2io5Write9write_all17h9469b32eef1f7433E"
 
 static Function *getFileWriteallDecl(Module *mod) {
   Function *func = mod->getFunction(MANGLED_IO_WRITE_WRITEALL_NAME);
-- 
2.25.1

